crypto/src/mls/conversation/decrypt.rs

```diff
@@ -295,10 +300,16 @@ impl MlsConversation {
                 ProcessMessageError::ValidationError(ValidationError::WrongEpoch) => {
                     if is_duplicate {
                         CryptoError::DuplicateMessage
-                    } else if msg_epoch == self.group.epoch().as_u64() + 1 {
+                    } else if msg_epoch == group_epoch + 1 {
                         // limit to next epoch otherwise if we were buffering a commit for epoch + 2
                         // we would fail when trying to decrypt it in [MlsCentral::commit_accepted]
                         CryptoError::BufferedFutureMessage
+                    } else if msg_epoch < group_epoch {
+                        match content_type {
+                            ContentType::Application => CryptoError::WrongEpoch,
```

Comment:
Shouldn't this be `MessageEpochTooOld`?

Reply:
@typfel Could be. But we were not returning MessageEpochTooOld before in this very specific case (we were returning WrongEpoch) so in the spirit of keeping things consistent, I kept it as-is

Reply:
Ok, thought we were changing the behavior. The it's fine.